### PR TITLE
golang: add error handling in simple consumer

### PR DESCRIPTION
**Pull request by Lmonster**
Add error handling in simple consumer in golang for Subscribe and Unsubscribe when getting available message queue from server

### CODE REVIEW

File: golang/simple_consumer.go [hunk 1]
```diff
@@ -142,7 +146,11 @@ func (sc *defaultSimpleConsumer) Subscribe(topic string, filterExpression *Filte
 }
 
 func (sc *defaultSimpleConsumer) Unsubscribe(topic string) error {
-	sc.cli.getMessageQueues(context.Background(), topic)
```
Comment (aaron-ai) [inline golang/simple_consumer.go:145]: Getting message queues is redundant for topic unsubscription?

**Review by drpmma — APPROVED** [review]
LGTM

File: golang/simple_consumer.go (hunk 1 above)
Comment (Lmonster) [reply golang/simple_consumer.go:145]: it makes sense, i'll update it with this pr

**Review by lollipopjin — APPROVED** [review]
LGTM

File: golang/simple_consumer.go (hunk 1 above)
Comment (Lmonster) [reply golang/simple_consumer.go:145]: > Getting message queues is redundant for topic unsubscription?

Done

**Comment by aaron-ai** [comment]
Thanks, @Lmonster !

**Pull request merged**